identity/backend/src/main/java/io/camunda/identity/usermanagement/model/User.java

```diff
@@ -5,11 +5,12 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.identity.user;
+package io.camunda.identity.usermanagement.model;
 
-public record CamundaUser(String username, boolean enabled) {
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 
-  public CamundaUser(final String username) {
-    this(username, true);
-  }
-}
+@Entity
+@Table(name = "users")
+public record User(@Id Long id, String username, String password, boolean enabled) {}
```

Comment:
❓ Is there any reason we use a record here vs a typical class in the other entities? I believe that the Profiles class could also be a record for example

Reply:
unfortunately Spring jpa is not record friendly and the reason it works with User is that we don't create or retrieve the entity from the repository. It's just there to avoid writing native queries.

Reply:
Ah ok